oauth2_http/java/com/google/auth/mtls/X509Provider.java

```diff
@@ -135,19 +130,19 @@ public KeyStore getKeyStore() throws IOException {
     } catch (Exception e) {
       // Wrap all other exception types to an IOException.
       throw new IOException(e);
-    } finally {
-      if (certStream != null) {
-        certStream.close();
-      }
-      if (privateKeyStream != null) {
-        privateKeyStream.close();
-      }
-      if (certAndPrivateKeyStream != null) {
-        certAndPrivateKeyStream.close();
-      }
     }
   }
 
+  @Override
+  public boolean isCertificateSourceAvailable() throws IOException {
```

Comment:
can this public method have javadocs. Same for the impl in SecureConnect

Reply:
Yup, will do.